pkg/config/config.go

```diff
@@ -0,0 +1,102 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package config
+
+import (
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"strings"
+	"time"
+
+	"github.com/api7/ingress-controller/pkg/types"
+	"gopkg.in/yaml.v2"
+)
+
+const (
+	_minimalResyncInterval = time.Second
```

Comment:
The Frequency  mini interval of resync 1 per second is too high, how about 1min, the same as default interval.

Reply:
I think 1min is too large, what about 30s.

Reply:
ok

Reply:
Changed.